src/pages/styles.css

```diff
@@ -9,6 +9,6 @@
   padding: 80px 40px;
   border: 1px solid rgba(113 119 144 / 25%);
   color: rgb(202, 202, 202);
-  margin: 0 auto;
+  margin: 0 auto 60px;
   position: relative;
 }
```

Comment:
1) The `.main` width is too small. Refer to design and set a correct width
2) You don't need margin here